### DISCUSSION TITLE
Option to set Max Parallel Running Workflows at a time

### DISCUSSION THREAD

**Discussion by ksharma24727** [elsa-workflows/elsa-core#3961] [Q&A]
Is there any option to give max parallel instance of a workflow.

Like we have a option to make a workflow singleton i.e. the workflow will run only once at a time.
Similarly is there any option to set this max instance of work flow to some number other than 1.

**Comment by sfmskywalker** [comment]
There's no such option for Elsa 2, but for Elsa 3 we probably could implement this option easily. Right now, Elsa 3 supports Singleton and a mode that limits workflow instances within the scope of a correlation ID. We could add a setting that allows you to specify the maximum number if instances too. Please feel free to open an issue for this.
